index.html

```diff
@@ -7,195 +7,157 @@
     <title>Portofolio</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
     <link rel="stylesheet"  href="style.css">
+    <link rel="stylesheet"  href="style-contact.css">
 </head>
 <body>
-       
-  <header>
-    <div class="my-logo"><a href="#">My Logo</a></div>
+       <header>
+    <a class="my-logo" href="#">My Logo</a>
     <nav class="my-menu">
      <img src="img/hamburger.svg" alt="my-menu icon">
     </nav>
   </header> 
-
-    <section id="introduction">
-      <h1>I’m Mladan, Glad to see you!</h1>
-      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-  <a>LET'S CONNECT</a>
+  <!--Section Introduction-->
+<section id="introduction">
+  <h1 class="im">I’m Mladan, Glad to see you!</h1>
+  <p class="soft-de">I’m a software developer! I can help you<br> build a product , feature or website Look<br>through some of my work and<br>experience! If you like what you see and<br> have a project you need coded, don’t<br> hestiate to contact me.</p>
+  <a class="let-con">LET'S CONNECT</a>

```

Comment:
kindly use some meaningful classes so that it is easier to read the code for other developers